vsphere/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,455 @@
+          not available, it will fallback to the VM name.
+          Use this if you install the agent in VMs so that the hosts are not duplicated
+          in the web application UI.
+        value:
+          type: boolean
+          default: false
+          example: false
+      - name: threads_count
+        required: false
+        description: |
+          This integration is using threads internally when making queries against the vCenter API.
+          Increasing this value will put more charge onto your vCenter server but may improve performances.
+        value:
+          type: integer
+          default: 4
+          example: 4
+      - name: excluded_host_tags
+        required: false
+        description: |
+          The vSphere integration collects tags for every ESXi host or VM in your environment.
+          Those tags can be `vcenter_server`, vsphere_type`, `vsphere_host`, `vsphere_folder`, `vsphere_cluster`
+          `vsphere_compute`, `vsphere_datacenter`, `vsphere_datastore` and are attached to the vSphere metrics by default.
+          By default those tags are submitted as host tags but you can submit them as metric tags if you need, see:
+          https://docs.datadoghq.com/tagging/using_tags/
+          You will lose the ability to filter your hosts on the tags you specify but they will appear faster
+          and will remove spikes from some `count by {TAG}` queries.
+          A good use case is to set this parameter to `[vsphere_host]`, indeed with VM migrations this tag is expected to
+          change often and it is usually helpful to be informed quickly of this change.
+        value:
+          type: array
+          items: 
+            type: string
+          default: []
+          example: 
+            - <HOST_TAG>
+      - name: metrics_per_query
+        required: false
+        description: |
+          This value is used to determine the number of metrics the check will retrieve in the same API call.
+          If too many API calls are made by the check, consider increasing this value
+          If payloads are too large, consider decreasing this value
+          A value <= 0 means unlimited: it makes one query for all the monitored objects
+        value:
+          type: integer
+          default: 500
+          example: 50
+      - name: max_historical_metrics
+        required: false
+        description: |
+          This value is used to determine the number of historical metrics the check will retrieve in the same API call.
+          Historical metrics collection is limited by the "config.vpxd.stats.maxQueryMetrics" configuration option
+          on vCenter, any API call requesting more metrics than this value will be denied.
+          By default, the check will retrieve the value of this parameter but you can override this behavior by specifying
+          a value here.
+        value:
+          type: integer
+          default: 200
```

Comment:
```suggestion
```

Using `example` will also set the default to the same value. You can use `default` in case you want default and example to be different.

Details: https://github.com/DataDog/integrations-core/blob/1f09e6137d02a06a17fb041312409ad9936be95b/datadog_checks_dev/datadog_checks/dev/tooling/configuration/consumers/example.py#L112-L132

Also, seems the value 200 was incorrect.